run_test.sh

```diff
@@ -215,19 +217,17 @@ run_with_expected_error() {
     local dkms_command="$3"
     local output_log=test_cmd_output.log
     local expected_output_log=test_cmd_expected_output.log
-    local error_code
+    local error_code=0
 
```

Comment:
Unless I'm missing something `run_with_expected_output()` can now become a simple wrapper around `run_with_expected_error()`, correct?

If so, feel free to fix that in a future PR.

Reply:
need to dig into that again at some point

Reply:
`run_with_expected_output` can just call `run_with_expected_error 0 "$@"`, prepended to next PR.